dotnet-client-libraries/YamlDotNet/Tests/ObjectGraphSerializeAndDeserializeTests.cs

```diff
@@ -0,0 +1,28 @@
+using App.Models;
+using App.UseCases;
+
+namespace Tests;
+
+public class ObjectGraphSerializeAndDeserializeTests
+{
+    [Fact]
+    public void WhenStoreIsSerialized_ThenYamlIsReturned()
+    {
+        var store = new Store { Name = "Test Store", Items = new List<Item>() };
+        var expectedYaml = "Name: Test Store\nItems: []\n";
```

Comment:
Same as with the test in the "Basic" tests, this one's failing for me due to the different newline characters.

```suggestion
        var expectedYaml = $"Name: Test Store{Environment.NewLine}Items: []{Environment.NewLine}";
```